src/utils/commonHelpers.js

```diff
@@ -0,0 +1,36 @@
+import {Listener} from "symbol-sdk";
+
+export class CommonHelpers {
+    /**
+     * Helper method to sleep for ms miliseconds
+     * @param {string} text
+     * @return {boolean}
+     */
+    static sleep(ms: number): Promise<void> {
+        return new Promise(resolve => {
+            setTimeout(() => {
+                resolve();
+            }, ms);
+        });
+    }
+
+    /**
+     * Helper method to retry opening websocket n times asynchronously
+     */
+    static async retryNTimes(listener: Listener, trials: number, interval: number) {
```

Comment:
IMO function name is not be clear enough. 

We can make the name more specefic:
`static async reconnectListener(listener: Listener, trials: number, interval: number) {` 

Or make the logic more generic to accept any callback:
`static async retryNTimes(callback: () => boolean, trials: number, interval: number) {`